extensions/wikia/AdEngine/js/video/player/porvata/googleImaPlayerFactory.js

```diff
@@ -28,6 +29,8 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory', [
 			mobileVideoAd = params.container.querySelector('video'),
 			eventListeners = {};
 
+		imaViewMode = win.google.ima.ViewMode;
```

Comment:
I don't really like we're keeping `imaViewMode` in global scope, I need a moment to rethink that